hedera-node/configuration/testnet/application.properties

```diff
@@ -4,7 +4,9 @@
 # It does not have to match bootstrap.properties once that file is deprecated by the switch from mono-service to hedera-config.
 
 #Overrides that differ based on the network
-ledger.id=0x01
-contracts.chainId=296
 bootstrap.genesisPublicKey=e06b22e0966108fa5d63fc6ae53f9824319b891cd4d6050dbf2b242be7e13344
+contracts.chainId=296
 contracts.knownBlockHash=
+ledger.id=0x01
+staking.rewardRate=273972602739726
```

Comment:
Where does `staking.rewardRate` come from?

Reply:
It was in the testnet config when devops pulled it this morning